pom.xml

```
@@ -35,32 +35,32 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-  <name>Confidence OpenFeature Provider</name>
+  <name>Confidence SDK</name>
   <organization>
       <name>com.spotify</name>
       <url>https://github.com/spotify</url>
   </organization>
 
   <issueManagement>
       <system>GitHub</system>
-      <url>https://github.com/spotify/confidence-openfeature-provider-java/issues</url>
+      <url>https://github.com/spotify/confidence-sdk-java/issues</url>
   </issueManagement>
 
   <licenses>
       <license>
           <name>Apache License 2.0</name>
-          <url>https://github.com/spotify/confidence-openfeature-provider-java/blob/main/LICENSE</url>
+          <url>https://github.com/spotify/confidence-sdk-java/blob/main/LICENSE</url>
           <distribution>repo</distribution>
       </license>
   </licenses>
 
   <scm>
-      <url>https://github.com/spotify/confidence-openfeature-provider-java</url>
+      <url>https://github.com/spotify/confidence-sdk-java</url>
       <connection>
-          scm:git:git@github.com:spotify/confidence-openfeature-provider-java.git
+          scm:git:git@github.com:spotify/sdk-java.git
```

I am not exactly sure what these scm uris are used for but they should probably be set to `confidence-sdk-java`
```suggestion
          scm:git:git@github.com:spotify/confidence-sdk-java.git
```